docs/UserGuide.md

```diff
@@ -625,3 +625,45 @@ You can delete all your customer and delivery data by calling this command._
 **After:**
 ![](images/clear_after.png)
 [&uarr; Back to Table of Contents](#table-of-contents)
+
+## Command Summary
+
+#### User
+
+| Command  | Format                                                                                                                       | Examples                                                                                                                    |
+|----------|------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Register | `register --user USERNAME --password PASSWORD --confirmPass CONFIRM_PASSWORD --secretQn SECRET_QUESTION --answer ANSWER`     | `register --user gabriel --password gabrielIsGreat --confirmPass gabrielIsGreat --secretQn First pet's name? --answer Koko` |
+| Login    | `login --user USERNAME --password PASSWORD`                                                                                  | `login --user gabriel --password gabrielIsGreat`                                                                            |
+| Update   | `update [--user USERNAME] [--password PASSWORD --confirmPass CONFIRM_PASSWORD] [--secretQn SECRET_QUESTION --answer ANSWER]` | `login --user gabrielV2 --password gabrielIsBest --confirmPass gabrielIsBest --secretQn Favourite Pet --answer BoBo`        |
+| Recover  | `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`                                   | `recover account --answer Koko --password newPassword123 --confirmPass newPassword123`                                      |
+| Logout   | `logout`                                                                                                                     | `logout`                                                                                                                    |
+
+#### Customer
```

Comment:
Should we arrange Customer and Delivery to be listed in the same order, e.g. Add, Find, List